lib/utils.js

```diff
@@ -23,6 +23,26 @@ var proxyaddr = require('proxy-addr');
 var qs = require('qs');
 var querystring = require('querystring');
 
+
+/**
+ * Helper function (requiring partial application)
+ * for setting exports.etag or exports.wetag.
+ *
+ * @param {Boolean} weak
+ * @return {Function}
+ * @api private
+ */
+
+function setTag(weak) {
```

Comment:
Please place the non-exported function with the others at the bottom of the file.